### DISCUSSION TITLE
IDs로 문제 검색하기 질문

### DISCUSSION THREAD

**Discussion by tjrkdgnl** [solvedac/unofficial-documentation#33] [질답] [answered]
*API 요구서 

1. 요구 기능
- Problems IDs Array를 파라미터로 전달하면 각 IDs에 대한 Problem의 정보를 담은 배열 리턴

2. 목적
- 앱을 만들려고 합니다. 자신이 푼 문제가 무엇인지 세부적으로 보여주는 기능을 만들고 싶은데 그럼 IDs마다 네트워크 호출이 일어나게 돼서 푼 문제만큼 네트워크 호출이 발생할 것 같습니다. 따라서 IDs를 담은 배열 한번만 전송하여 각 problems 정보를 담은 배열을 리턴 받고싶습니다.  가능할까요?

**Comment by RanolP** [comment]
본 저장소의 유지보수자는 solved.ac 서비스를 개발하는 사람이 아닙니다. 기능 추가의 경우 solved.ac 디스코드나 기타 연락책을 통해 shiftpsh님께 문의해주시기 바랍니다.

한편, 고급 검색 기능을 통해 `id:1000 | id:1002 | id:1007` 같은 쿼리를 날릴 경우 유사한 기능을 구현할 수 있습니다. 자세한 고급 검색 기능 설명은 https://solved.ac/search 를 참고해주세요.

**Comment by RanolP** [comment, marked as answer]
어 근데 옛날 API 찾아서 좀 고쳐보니 되긴 하네요 https://solved.ac/api/v3/problem/lookup?problemIds=1000,1002,1007 🤔

**Comment by tjrkdgnl** [comment]
먼저 잘 모르고 요구사항을 남겨서 죄송합니다 ㅜㅜ 그럼 말씀해주신 것처럼 사용하면 될까요?

**Comment by RanolP** [comment]
네네

**Comment by tjrkdgnl** [comment]
감사합니다. document에서 테스트했고 잘 되는 것 확인했습니다!!!
